includes/class-wc-payments-account.php

```diff
@@ -519,8 +521,13 @@ public function maybe_redirect_to_capital_offer() {
 		$refresh_url = add_query_arg( [ 'wcpay-loan-offer' => '' ], admin_url( 'admin.php' ) );
 
 		try {
-			$capital_link = $this->payments_api_client->get_capital_link( 'capital_financing_offer', $return_url, $refresh_url );
+			$request = Get_Account_Capital_Link::create();
+			$type    = 'capital_financing_offer';
+			$request->set_type( $type );
+			$request->set_redirect_url( $return_url );
+			$request->set_refresh_url( $refresh_url );
 
+			$capital_link = $request->send( 'wcpay_get_account_capital_link', $type, $return_url, $refresh_url );
```

Comment:
Small nitpick: You don't have to send these parameters into the filter since you will be able to fetch them from the request class, but yet again, it is nice to have them. Feel free to leave them here or dismiss them.

Reply:
TIL and make a lot of sense. Removing these args now 5243feab0af068c33ac65877689e9cb5b14bb871